inst/stan/wwinference.stan

```diff
@@ -35,15 +35,14 @@ data {
   vector<lower = 1e-20>[n_subpops] subpop_size; // the population sizes for each subpopulation
   real<lower = state_pop> norm_pop;
   array[owt] int<lower=1,upper=ot + ht> ww_sampled_times; // a list of all of the days on which WW is sampled
-                                   // will be mapped to the corresponding sites (ww_sampled_sites)
+                                   // will be mapped to the corresponding subpops (ww_sampled_subpops)
   array[oht] int<lower=1, upper=ot> hosp_times; // the days on which hospital admissions are observed
-  array[owt] int<lower=1,upper=n_subpops> ww_sampled_sites; // vector of unique sites in order of the sampled times
-  array[owt] int<lower=1,upper=n_ww_lab_sites> ww_sampled_lab_sites; // vector of unique lab-site combos i
-   	     // n order of the sampled times
+  array[owt] int<lower=1,upper=n_subpops> ww_sampled_subpops; // vector of unique subpops in order of the sampled times
+  array[owt] int<lower=1,upper=n_ww_lab_sites+1> ww_sampled_lab_sites; // vector mapping the subpops to lab-site combos
```

Comment:
Presumably the `n_ww_lab_sites+1`th index corresponds to an auxiliary site without observed wastewater. But given that this vector associates wastewater observations to lab_site combos, that index should never be used and should raise an error if passed in.

```suggestion
  array[owt] int<lower=1,upper=n_ww_lab_sites> ww_sampled_lab_sites; // vector mapping the subpops to lab-site combos
```

Reply:
Oh yes sorry this was an artifact from the way I was setting it up initially, then changed things around